src/main/java/no/finn/unleash/DefaultUnleash.java

```diff
@@ -92,6 +92,10 @@ public boolean isEnabled(final String toggleName, final UnleashContext context ,
         return enabled;
     }
 
+    public FeatureToggle getFeatureToggleDefinition(String toggleName) {
+        return toggleRepository.getToggle(toggleName);
```

Comment:
then it will return null, could preferably be wrapped in to an optional perhaps?